omega/omega-transport/omega-transport-feign/src/main/java/org/apache/servicecomb/pack/omega/transport/feign/FeignAutoConfiguration.java

```diff
@@ -29,8 +28,7 @@
 
     @Bean
     @ConditionalOnClass(RequestInterceptor.class)
-    @ConditionalOnBean(OmegaContext.class)
-    public RequestInterceptor feignClientRequestInterceptor(OmegaContext omegaContext){
-        return new FeignClientRequestInterceptor(omegaContext);
+    public RequestInterceptor feignClientRequestInterceptor(){
+        return new FeignClientRequestInterceptor(OmegaContextManager.getContext());
```

Comment:
It‘s better to inject the OmegaContext instead of get from the OmegaContextManager.

Reply:
OmegaConetxt was not managered by spring container，it should use getContext method instead of autowire